demos/tests/cases.py

```diff
@@ -378,6 +378,15 @@ def single_option_cases(key, *args):
     )),
 
     # TODO: object_detection_demo_yolov3_async: no models.lst
+    PythonDemo(subdirectory='object_detection_demo_yolov3_async', device_keys=['-d'], test_cases=combine_cases(
+        TestCase(options={'--no_show': None,
+                          '-i': DataPatternArg('object-detection-demo-ssd-async')}),
+        single_option_cases('-m',
+                            ModelArg('yolo-v1-tiny-tf'),
+                            ModelArg('yolo-v2-tiny-tf'),
+                            ModelArg('yolo-v2-tf'),
+                            ModelArg('yolo-v3-tf')),
```

Comment:
The rest of the file follows different indent style. I think you should stick to it.

Reply:
This is also applicable for `'-i': DataPatternArg('object-detection-demo-ssd-async')}),`